pythonforandroid/recipes/pygame2/__init__.py

```diff
@@ -0,0 +1,57 @@
+from pythonforandroid.recipe import CompiledComponentsPythonRecipe
+from os.path import join
+from pythonforandroid.toolchain import current_directory
+
+
+class Pygame2Recipe(CompiledComponentsPythonRecipe):
+
+    version = '2.0.0-dev7'
+    url = 'https://github.com/pygame/pygame/archive/android-2.0.0-dev7.tar.gz'
+
+    site_packages_name = 'pygame'
+    name = 'pygame2'
```

Comment:
Are you going to officially make the new release pygame2, with e.g. a different name in pip? If so then fine, but if not it's probably more obvious to just call the recipe "pygame". Happy to go with your preference though.